terraform/main.tf

```diff
@@ -40,6 +40,7 @@ module "postgresql" {
   project                     = var.project_prefix
   rds_backup_retention_period = var.backup_retention_period
   rds_db_name                 = "geostore"
+  rds_version                 = "12.8"
```

Comment:
I see you parameterised this at the postgresql module level but hard-coded it at the top level here. If the intention is to enable environment specific version (saw that in commit message unless I misunderstood), we should parameterise it at this level as well.